owl/SOMA-OBJ.owl

```diff
@@ -24,6 +24,44 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA-OBJ.owl#hasNetForce -->
+
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/SOMA-OBJ.owl#hasNetForce">
+        <rdfs:subPropertyOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#hasRegion"/>
+        <owl:inverseOf rdf:resource="http://www.ease-crc.org/ont/SOMA-OBJ.owl#isNetForceOf"/>
+        <rdfs:domain rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#PhysicalObject"/>
+        <rdfs:range rdf:resource="http://www.ease-crc.org/ont/SOMA-OBJ.owl#NetForce"/>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">The accumulated force acting upon an object.</rdfs:comment>
```

Comment:
Perhaps something like "A relation between a physical object and the total force acting on it."

Reply:
ok!